packages/gatsby/src/commands/build-html.ts

```diff
@@ -14,6 +15,23 @@ import { IProgram, Stage } from "./types"
 type IActivity = any // TODO
 type IWorkerPool = any // TODO
 
+export interface IWebpackWatchingPauseResume extends webpack.Watching {
+  suspend: () => void
+  resume: () => void
+}
+
+let devssrWebpackCompilier: webpack.Compiler
```

Comment:
typo? (and need search'n'replace across few files :) )
```suggestion
let devssrWebpackCompiler: webpack.Compiler
```

Reply:
heh